src/js/cookieMessage.js

```diff
@@ -2,6 +2,45 @@ class CookieMessage {
 
 	constructor (CookieMessageEl) {
 		this.CookieMessageEl = CookieMessageEl;
+
+		return this.setup();
```

Comment:
it shouldn't return anything, just have it execute `this.setup():` as it returns `this` as a default which in this case will be the object for the instance that executed the constructor